app/assets/javascripts/discourse/tests/integration/components/topic-participant-test.js

```diff
@@ -0,0 +1,49 @@
+import { render } from "@ember/test-helpers";
+import { hbs } from "ember-cli-htmlbars";
+import { module, test } from "qunit";
+import { setupRenderingTest } from "discourse/tests/helpers/component-test";
+import { exists } from "discourse/tests/helpers/qunit-helpers";
+
+module("Integration | Component | topic-participant", function (hooks) {
+  setupRenderingTest(hooks);
+
+  test("one post", async function (assert) {
+    this.set("args", {
+      username: "test",
+      avatar_template: "/images/avatar.png",
+      post_count: 1,
+    });
+
+    await render(hbs`<TopicMap::TopicParticipant @participant={{this.args}}/>`);
+
+    assert.ok(exists("a.poster.trigger-user-card"));
```

Comment:
This is fine, but you might find it easier to use [qunit-dom](https://github.com/mainmatter/qunit-dom/blob/master/API.md) when checking the DOM. So this line would be

```js
assert.dom("a.poster.trigger-user-card").exists()
```